src/ddmd/attrib.d

```diff
@@ -165,6 +165,9 @@ extern (C++) abstract class AttribDeclaration : Dsymbol
 
     override void semantic(Scope* sc)
     {
+        if (semanticRun != PASSinit)
```

Comment:
Isn't semantic() supposed to be called recursively here until `semanticRun >= PASSsemanticdone`?